### PR TITLE
Faqs for common errors when upgrading Android SDK

### PR DESCRIPTION
**Pull request by praveek**
…docs into staging

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/faq.md [hunk 1]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+
+Android SDK now supports a minimum API level of **19**.  If your application targets a lower API level, you will see the following build failure:
+
+```
+Manifest merger failed : uses-sdk:minSdkVersion 14 cannot be smaller than version 19 declared in library [com.adobe.marketing.mobile:core:2.0.0]
+```
+
+To fix this build failure, increase the minSdkVersion for your Android project to **19** or above.
+
+### When I add Android SDK my project, why do i get an error about invoke-custom support and enabling desugaring?
```
Comment (swarna04) [inline src/pages/documentation/faq.md:124]: When I add Android SDK **to** my project...

File: src/pages/documentation/faq.md [hunk 2]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+
+See https://developer.android.com/studio/write/java8-support.html for details.
+Alternatively, increase the minSdkVersion to 26 or above.
+```
+
+To fix this build failure, you can follow one of two options:
+- Add the listed compileOptions from the error message to your app-level build.gradle file.
+- Increase the minSdkVersion for your Android project to 26 or above.
+
+### Why do i see a warning in AndroidManifest.xml about missing 'com.adobe.marketing.mobile.FullscreenMessageActivity' class?
```
Comment (swarna04) [inline src/pages/documentation/faq.md:147]: letter case: Why do **I**...

File: src/pages/documentation/faq.md [hunk 3]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+
+After upgrading to latest Android SDK, you will see the following build warning if your application previously set up in-app messages with Campaign Standard. 
+
+```
+Class referenced in the manifest, `com.adobe.marketing.mobile.FullscreenMessageActivity`, was not found in the project or the libraries
+
+Unresolved class 'FullscreenMessageActivity'
+```
+
+To resolve the build warning, remove FullscreenMessageActivity from your application's manifest file. Campaign Standard no longer requires appliction to add **FullscreenMessageActivity** to their manifest.
```
Comment (swarna04) [inline src/pages/documentation/faq.md:157]: typo: Campaign Standard SDK no longer requires **application**...

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Review by swarna04 — APPROVED** [review]

File: src/pages/documentation/faq.md [hunk 4]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+
+After upgrading to latest Android SDK, you will see the following build warning if your application previously set up in-app messages with Campaign Standard. 
+
+```
+Class referenced in the manifest, `com.adobe.marketing.mobile.FullscreenMessageActivity`, was not found in the project or the libraries
+
+Unresolved class 'FullscreenMessageActivity'
+```
+
+To resolve the build warning, remove FullscreenMessageActivity from your application's manifest file. Campaign Standard SDK no longer requires appliction to add **FullscreenMessageActivity** to their manifest.
```
Comment (swarna04) [inline src/pages/documentation/faq.md:157]: typo: appliction -> application is still present

Comment (praveek) [reply src/pages/documentation/faq.md]: oops. missed it.

File: src/pages/documentation/faq.md [hunk 5]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 }
 ```
 
+### Is there a change in minimum API level supported by Android SDK?
```
Comment (emdobrin) [inline src/pages/documentation/faq.md:114]: Can we rename all the `Android SDK` occurrences to one of these options: 
`Android Platform Mobile SDK`, `Android Mobile SDK`, `Mobile SDK for Android`

**Review by emdobrin — COMMENTED** [review]
Just a small renaming comment, looks great otherwise

File: src/pages/documentation/faq.md [hunk 6]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+Alternatively, increase the minSdkVersion to 26 or above.
+```
+
+To fix this build failure, you can follow one of two options:
+- Add the listed compileOptions from the error message to your app-level build.gradle file.
+- Increase the minSdkVersion for your Android project to 26 or above.
+
+### Why do I see a warning in AndroidManifest.xml about missing 'com.adobe.marketing.mobile.FullscreenMessageActivity' class?
+
+After upgrading to latest Android SDK, you will see the following build warning if your application previously set up in-app messages with Campaign Standard.
```
Comment (calebk1) [inline src/pages/documentation/faq.md:149]: "... upgrading to the latest version of the Mobile SDK for Android, you will..."

**Review by calebk1 — COMMENTED** [review]
Just a couple of comments!

File: src/pages/documentation/faq.md [hunk 7]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 }
 ```
 
+### Is there a change in minimum API level supported by Mobile SDK for Android?
+
+Mobile SDK for Android now supports a minimum API level of **19**.  If your application targets a lower API level, you will see the following build failure:
```
Comment (calebk1) [inline src/pages/documentation/faq.md:116]: Very minor, but a double space between the period and if.

File: src/pages/documentation/faq.md [hunk 8]
```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
...
+	}
+}
+
+See https://developer.android.com/studio/write/java8-support.html for details.
+Alternatively, increase the minSdkVersion to 26 or above.
+```
+
+To fix this build failure, you can follow one of two options:
+- Add the listed compileOptions from the error message to your app-level build.gradle file.
+- Increase the minSdkVersion for your Android project to 26 or above.
```
Comment (calebk1) [inline src/pages/documentation/faq.md:145]: Also minor, but since we bolded 19 earlier, we should bold the 26 here for parallelization.

**Review by calebk1 — APPROVED** [review]
Looks good! Just to very small fixes, and we can be good to merge.

**Pull request merged**